packages/interactive_media_ads/lib/interactive_media_ads.dart

```diff
@@ -5,6 +5,15 @@
 export 'src/ad_display_container.dart';
 export 'src/ads_loader.dart';
 export 'src/ads_manager_delegate.dart';
-export 'src/platform_interface/ad_error.dart';
-export 'src/platform_interface/ad_event.dart';
-export 'src/platform_interface/ads_request.dart';
+export 'src/android/android_interactive_media_ads.dart'
+    show AndroidInteractiveMediaAds;
+export 'src/platform_interface/platform_interface.dart'
+    show
```

Comment:
In some cases in the past we've regretted directly re-exposing platform interface APIs in the app-facing API (instead of having a second version and converting); have you audited these to make sure we're very unlikely to ever want to make any breaking changes to them, such as adding a required constructor param, or in having some method that implementations need but clients should never use?

Reply:
Does the fact that the platform interface, the implementations, and the app-facing API live in the same package affect this? This should ideally give us more flexibility to make private methods/constructors/subclasses if we need to.

Reply:
I guess we could always move these classes to the app-facing directory and make a new class that the platform interface code used, if we need to make breaking changes later, since changing the platform interface wouldn't be public in this structure.